tinyfilemanager.php

```diff
@@ -1783,7 +1783,7 @@ class="edit-file"><i class="fa fa-pencil-square-o"></i> <?php echo lng('Advanced
             } elseif ($is_image) {
                 // Image content
                 if (in_array($ext, array('gif', 'jpg', 'jpeg', 'png', 'bmp', 'ico', 'svg', 'webp', 'avif'))) {
-                    echo '<p><img src="' . fm_enc($file_url) . '" alt="image" class="preview-img-container" class="preview-img"></p>';
+                    echo '<p><img src="' . fm_enc($file_url) . '" alt="" class="preview-img"></p>';
```

Comment:
Why change the `alt` tag? For users relying on screen readers or other assistive technologies, the alt attribute provides an alternative textual description. When the alt attribute is empty, it tells the screen reader to ignore the image, treating it as decorative or non-essential. A screen reader may choose to hide an image with an empty alt tag instead of, say, zooming in on it to present it to the user.

You could argue "image" is useless, but leaving it blank essentially turns off accessibility for the image. If you're just trying to clean up the code in the line you were editing, you could use `basename($file_url)`. Which I think would be a great idea, but your patch isn't about that so I'd probably just leave it as "image".

Reply:
Thanks for the knowledge.. I already told in my comment that I am Kinda new to all this. So cut me some Slack. So What are your opinions on how :
> Images not fit the size of devices and were not viewable have to scroll down and up to view the whole big image

How should we solve that and also keep `alt="image" class="preview-img-container"` as you said in your comment. I think if you have figured it out, you would have already posted the fix. 
That was my real intension so that the developer will look for a fix for this to solve. Anyways if you have the solution do post it so that developer can put that tin Code.